docs: simplify component descriptions (#294)
arcus-azure/arcus: website/resources/components/background-jobs.md

---
title: Arcus Background Jobs

> description: Strengthen you application with commonly repeated tasks ranging from automatic secret invalidation of Azure Key Vault secrets to publish an event on Azure EventGrid and more.

Reviewer: ```suggestion
description: Strengthen your application with commonly repeated tasks ranging from automatic secret invalidation of Azure Key Vault secrets to publish an event on Azure EventGrid and more.
```

Reviewer 2: ```suggestion
description: Strengthen your application with commonly repeated tasks ranging from automatic secret invalidation of Azure Key Vault secrets to publishing events on Azure EventGrid and more.
```
